### DISCUSSION TITLE
Classic defaults during setup

### DISCUSSION THREAD

**Discussion by Shavixinio** [ppy/osu#30297] [Q&A] [answered]
During the setup, if you choose to have the "classic defaults", it sets the Hold-to-confirm activation time to 0 ms, which means if you hit the tilde key, the beatmap instantly retries, but at the same time it doesn't match stable, because in stable the beatmap retry confirmation time is around 100-150 ms iirc. So if you want to match the real stable hold-to-confirm activation time, you'd have to apply the new default activation time, but it also conflicts with the escape activation time being longer, which also doesn't match stable.

I believe some changes should be made to this system, such as potentially making a separate slider or toggle for changing the retry button's activation time, and another one for the escape button or other things. If a player coming from stable chooses the classic defaults, there's a chance they could accidentally retry a beatmap instead of pausing it because of bad finger placement, which could end up in them losing a lot of progress, especially if the player is in the middle of a good run, which did happen to me a few times, potentially wasting the effort.

**Comment by SchiavoAnto** [comment, marked as answer]
The hold-to-activate time for pausing has been removed in #30309.
Regarding quick-retry, probably should be changed to the same value as stable (using classic defaults).

**Reply by Shavixinio** [reply]
Happy to hear that!
